### ISSUE TITLE
[A11y] Backend: Manage oEmbeds

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3359]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Reference Link: https://oregondigital.org/oembeds?locale=en

Logged in gray sidebar Tasks > Manage oEmbeds

Recieved the following errors from Accessibility Insights, same issue for 410 errors

<img width="1094" alt="Image" src="https://github.com/user-attachments/assets/08f97daa-6d52-4588-beaa-ea628cc969ae" />

### Suggested Fix

Fix any of the following:

Element has insufficient color contrast of 2.48 (foreground color: #ffffff, background color: #5cb85c, font size: 9.0pt (12px), font weight: bold). Expected contrast ratio of 4.5:1

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Revalidate the page using accessibility insights.
